tests/runtime/main.rs

```diff
@@ -115,15 +115,19 @@ fn tests(name: &str, dir_name: &str) -> Result<Vec<PathBuf>> {
     let mut c = Vec::new();
     let mut java = Vec::new();
     let mut go = Vec::new();
-    let mut c_sharp = Vec::new();
+    let mut c_sharp: Vec<PathBuf> = Vec::new();
     for file in dir.read_dir()? {
         let path = file?.path();
         match path.extension().and_then(|s| s.to_str()) {
             Some("c") => c.push(path),
             Some("java") => java.push(path),
             Some("rs") => rust.push(path),
             Some("go") => go.push(path),
-            Some("cs") => c_sharp.push(path),
+            Some("cs") => {
+                if cfg!(windows) {
+                    c_sharp.push(path);
+                }
```

Comment:
Question: When mono is added we will want to still add this for the linux case?  Since we aren't running the tests on linux right now, is this needed?

Reply:
The problem is the yml matrix in `build.yml` tries to run all the OSes and if it creates the c# files then it will fail to compile them.  Agree we will have to revisit this for Mono, not sure about MacOS, but for Linux we will want it.